api/tests/acceptance/application/competences/competences_test.js

```diff
@@ -787,7 +787,7 @@ describe('Acceptance | Route | competences', () => {
           .get('/v0/airtableBaseValue/Competences/recCompetence404')
           .query({})
           .matchHeader('authorization', 'Bearer airtableApiKeyValue')
-          .reply(404, airtableCompetence);;
+          .reply(404, airtableCompetence);
```

Comment:
```suggestion
          .reply(404);
```

Si on a une 404, pourquoi on aurait le `airtableCompetence` de renvoyé ? 🤔